metricflow-semantics/metricflow_semantics/specs/where_filter/where_filter_spec.py

```diff
@@ -61,7 +66,7 @@ def merge(self, other: WhereFilterSpec) -> WhereFilterSpec:  # noqa: D102
         return WhereFilterSpec(
             where_sql=f"({self.where_sql}) AND ({other.where_sql})",
             bind_parameters=self.bind_parameters.combine(other.bind_parameters),
-            linkable_specs=ordered_dedupe(self.linkable_specs, other.linkable_specs),
```

Comment:
Does the ordering matter from the previous code? Just checking since it looks like it was removed in the new code

Reply:
The `.dedupe()` call is actually ordered, so it should be the same.